include/compose.bash

```diff
@@ -216,6 +216,9 @@ cloudbreak:
         - CB_SMTP_SENDER_FROM=$CLOUDBREAK_SMTP_SENDER_FROM
         - CB_BAYWATCH_ENABLED=$CB_BAYWATCH_ENABLED
         - CB_BAYWATCH_EXTERN_LOCATION=$CB_BAYWATCH_EXTERN_LOCATION
+        - CB_CERT_DIR=/certs
```

Comment:
I am not sure that we need these hard coded values since the same variables are set up in java code